src/test/java/fm/last/moji/integration/AbstractMojiIT.java

```diff
@@ -133,4 +133,62 @@ String newKey() {
     return newKey(RandomStringUtils.randomAlphanumeric(16));
   }
 
+  private void initMogileTestData() throws IOException {
+    if(initMogileTestDataIsDone) {
```

Comment:
Are you trying to ensure this method is only called *once*? If so you could just annotate it with @BeforeClass and not call it from @Before, I think that will get the behaviour you are trying to achieve without having to do this yourself.